app/support/auto_logout.rb

```diff
@@ -0,0 +1,38 @@
+class AutoLogout
```

Comment:
I realize this class isn't "yours" but there are parts of this extraction that could use cleanup (despite Code Climate giving it an "A").